lib/meta_inspector/parsers/images.rb

```diff
@@ -1,7 +1,9 @@
+require 'fastimage'
+
 module MetaInspector
   module Parsers
     class ImagesParser < Base
-      delegate [:parsed, :meta, :base_url]         => :@main_parser
```

Comment:
This kind of whitespacing is intented, to align similar lines so that hash rockets are on the same column. Makes it a bit easier to read.